roxie/udplib/udptrr.cpp

```diff
@@ -469,81 +417,96 @@ class CReceiveManager : implements IReceiveManager, public CInterface
             adjustPriority(1);
         #endif
             UdpRequestToSendMsg msg;
-            unsigned timeoutExpires = msTick() + 5000;
             while (running)
             {
+                unsigned timeout = 5000;
```

Comment:
should be a symbolic constant

Reply:
Should be configurable

Reply:
Well, maybe. It doesn't really matter what this value is (unless something goes wrong with the logic!)